griptape/artifacts/base_artifact.py

```diff
@@ -15,6 +14,20 @@
 
 @define
 class BaseArtifact(SerializableMixin, ABC):
+    """Serves as the base class for all Artifacts.
+
+    Artifacts are used to store data that can be provided as input to, or received as output from, a language model (LLM).
```

Comment:
We change this definition, right? Aren't they more like general containers for data + metadata now, right?